setup.py

```diff
@@ -25,85 +25,116 @@
 import six
 
 if six.PY3:
-    exec(open('ipopt/version.py', encoding="utf-8").read())
+    exec(open("cyipopt/version.py", encoding="utf-8").read())
 else:
-    exec(open('ipopt/version.py').read())
+    exec(open("cyipopt/version.py").read())
 
-PACKAGE_NAME = 'ipopt'
+PACKAGE_NAME = "cyipopt"
+DEPRICATED_PACKAGE_NAME = "ipopt"
 VERSION = __version__
-DESCRIPTION = 'A Cython wrapper to the IPOPT optimization package'
+DESCRIPTION = "A Cython wrapper to the IPOPT optimization package"
 if six.PY3:
-    with open('README.rst', encoding="utf-8") as f:
+    with open("README.rst", encoding="utf-8") as f:
         LONG_DESCRIPTION = f.read()
 else:
-    with open('README.rst') as f:
+    with open("README.rst") as f:
         LONG_DESCRIPTION = f.read()
-AUTHOR = 'Matthias Kümmerer'
-EMAIL = 'matthias.kuemmerer@bethgelab.org'
+KEYWORDS = [
+    "optimization",
+    "nonlinear programming",
+    "ipopt",
+    "interior-point",
+    "nlp",
+    "coin-or",
+]
+AUTHOR = "Matthias Kümmerer"
+EMAIL = "matthias.kuemmerer@bethgelab.org"
 URL = "https://github.com/matthias-k/cyipopt"
-DEPENDENCIES = ['numpy', 'cython', 'six', 'future', 'setuptools']
+DEPENDENCIES = [
+    "numpy",
+    "cython",
+    "six",
+    "future",
+    "setuptools",
+]
```

Comment:
We should pick some minimum version numbers for these. I often choose those that are in the latest Ubuntu LTS release.

Reply:
Thanks very much, I never know how to bound these so I like your suggestion to use Ubuntu as a reference for choosing.

Reply:
The only issue with the LTS release is that after two years, it's hard to keep the package compatible with packages that old. I guess just following the 6 month Ubuntu/Debian releases might be a better practice, or every other release so your software works with 1 year old dependencies at the most.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/8ac7faf4e399ddd2441be00bad8a20e140902c7c.